File: app/src/main/java/com/pdg/androidbasics/model/Item.java
package com.pdg.androidbasics.model;

public class Item {
    int id;
    String name;
    String subtitle;
    String image;
    float price;
    String description;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getSubtitle() {
        return subtitle;
    }

    public String getImage() {
        return image;
    }

    public float getPrice() {
        return price;
    }

    public String getDescription() {
        return description;
    }

    public Item(int id,
                String name,
                String subtitle,
                String image,
                float price,
                String description) {
        this.id = id;
        this.name = name;
        this.subtitle = subtitle;
        this.image = image;
        this.price = price;
        this.description = description;

    }
}

File: app/src/main/java/com/pdg/androidbasics/model/Item.kt
package com.pdg.androidbasics.model

class Item(
    id: Int,
    name: String,
    subtitle: String,
    image: String,
    price: Float,
    description: String
) {
    var id: Int = 0
        internal set
    var name: String
        internal set
    var subtitle: String
        internal set
    var image: String
        internal set
    var price: Float = 0.toFloat()
        internal set
    var description: String
        internal set

    init {
        this.id = id
        this.name = name
        this.subtitle = subtitle
        this.image = image
        this.price = price
        this.description = description

    }
}
